frontend/src/employee-frontend/components/IncomeStatementPage.tsx

```diff
@@ -0,0 +1,357 @@
+            ? i18n.incomeStatement.incomesRegister
+            : i18n.incomeStatement.attachmentsAndKela
+        }
+      />
+      {gross.estimatedIncome !== null && (
+        <Row
+          label={i18n.incomeStatement.grossEstimatedIncome}
+          value={<EstimatedIncome estimatedIncome={gross.estimatedIncome} />}
+        />
+      )}
+      <Row
+        label={i18n.incomeStatement.otherIncome}
+        value={
+          <>
+            {gross.otherIncome.map((incomeType) => (
+              <Item key={incomeType}>
+                {i18n.incomeStatement.otherIncomeTypes[incomeType]}
+              </Item>
+            ))}
+          </>
+        }
+      />
+    </>
+  )
+}
+
+function EstimatedIncome({
+  estimatedIncome
+}: {
+  estimatedIncome: EstimatedIncome
+}) {
+  const { i18n } = useTranslation()
+  return (
+    <FixedSpaceRow>
+      <FixedSpaceColumn>
+        <Label>{i18n.incomeStatement.estimatedMonthlyIncome}</Label>
+        <div>{estimatedIncome.estimatedMonthlyIncome}</div>
+      </FixedSpaceColumn>
+      <FixedSpaceColumn>
+        <Label>{i18n.incomeStatement.timeRange}</Label>
+        <div>
+          {estimatedIncome.incomeStartDate.format()} -{' '}
+          {estimatedIncome.incomeEndDate?.format()}
+        </div>
+      </FixedSpaceColumn>
+    </FixedSpaceRow>
+  )
+}
+
+function EntrepreneurIncome({ entrepreneur }: { entrepreneur: Entrepreneur }) {
+  const { i18n } = useTranslation()
+  const yesno = makeYesNo(i18n)
+
+  return (
+    <>
+      <H2>{i18n.incomeStatement.entrepreneurTitle}</H2>
+      <Row
+        label={i18n.incomeStatement.fullTimeLabel}
+        value={
+          entrepreneur.fullTime
+            ? i18n.incomeStatement.fullTime
+            : i18n.incomeStatement.partTime
+        }
+      />
+      <Row
+        label={i18n.incomeStatement.startOfEntrepreneurship}
+        value={entrepreneur.startOfEntrepreneurship.format()}
+      />
+      <Row
+        label={i18n.incomeStatement.spouseWorksInCompany}
+        value={yesno(entrepreneur.spouseWorksInCompany)}
+      />
+      <Row
+        label={i18n.incomeStatement.startupGrant}
+        value={yesno(entrepreneur.startupGrant)}
+      />
+      <Row
+        label={i18n.incomeStatement.kelaInspectionConsent}
+        value={yesno(true)}
+      />
+      <H3>{i18n.incomeStatement.companyInfoTitle}</H3>
+      <Row
+        label={i18n.incomeStatement.companyType}
+        value={
+          <>
+            {entrepreneur.selfEmployed !== null && (
+              <Item>{i18n.incomeStatement.selfEmployed}</Item>
+            )}
+            {entrepreneur.limitedCompany !== null && (
+              <Item>{i18n.incomeStatement.limitedCompany}</Item>
+            )}
+            {entrepreneur.limitedCompany !== null && (
+              <Item>{i18n.incomeStatement.partnership}</Item>
+            )}
+            {entrepreneur.limitedCompany !== null && (
+              <Item>{i18n.incomeStatement.lightEntrepreneur}</Item>
```

Comment:
These three null checks need some love.

Moreover, a simple truthiness check should suffice instead of `!== null` where checking for object.